### ISSUE TITLE
Minus sign disappears on focus (and blur) 

### ISSUE THREAD

**Issue by glenncoppens** [dm4t2/vue-currency-input#123]
Hi @dm4t2 ,

## Description
It seems like there's an edge-case issue when parsing negative values with the following config (see your VCurrencyField.vue component): 
[![Edit Using Vue Currency Input with Vuetify](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/using-vue-currency-input-with-vuetify-8g4do?fontsize=14&hidenavigation=1&theme=dark) 

### Doesn't work:
```
locale:  'nl',
distractionFree: true,
allowNegative: true
```
### Does work:
```
locale:  'nl',
distractionFree: false,
allowNegative: true
```
```
locale:  'us',
distractionFree: true,
allowNegative: true
```

## Current workaround
* disabling option `"distractionFree"` to `false` :)

## Other
Anyhow, thanks for this little piece of magic, nice job! 🥇

**Comment by dm4t2** [comment]
Hi, thanks for reporting. This is definitely a bug. I will look at it.

**Comment by dm4t2** [comment]
Fixed in [Release 1.19.0](https://github.com/dm4t2/vue-currency-input/releases/tag/1.19.0) 🎉
